_includes/code/blackjack.cc

```diff
@@ -0,0 +1,69 @@
+#include <queue>
+
+
+using namespace std;
+
+// Suits
+int spades = 0;
+int hearts = 1;
+int diamonds = 2;
+int clubs = 3;
```

Comment:
I'm unsure. On the one hand, it does add type safety. On the other hand, I don't want to add explaining what an enum is to this challenge. Your call, I think either way works